ui/js/group.js

```diff
@@ -160,29 +160,40 @@ require(['jquery', 'oae.core'], function($, oae) {
 
         // Structure that will be used to construct the left hand navigation pages
         var lhNavPages = [];
-        // Only show the recent activity to group members
-        if (groupProfile.isMember) {
-            lhNavPages.push({
-                'id': 'activity',
-                'title': oae.api.i18n.translate('__MSG__RECENT_ACTIVITY__'),
-                'icon': 'fa-tachometer',
-                'closeNav': true,
-                'layout': [
-                    {
-                        'width': 'col-md-12',
-                        'widgets': [
-                            {
-                                'name': 'activity',
-                                'settings': {
-                                    'context': groupProfile,
-                                    'canManage': groupProfile.isManager
-                                }
+
+        // Show the activity and about widgets on the main page
+        lhNavPages.push({
+            'id': 'activity',
+            'title': oae.api.i18n.translate('__MSG__RECENT_ACTIVITY__'),
+            'icon': 'fa-tachometer',
+            'closeNav': true,
+            'layout': [
+                {
+                    'width': 'col-md-9',
```

Comment:
@mrvisser : Would you be able to change this to `col-md-8 col-lg-9` for the recent activity widget and `col-md-4 col-lg-3` for the group profile widget? The current implementation is slightly too narrow at a narrow md width.